### PR TITLE
Feature/legionairre nuc230 transfer frame component

### PR DESCRIPTION
**Pull request by legionairre**
localhost_8080_components_frame html

### CODE REVIEW

File: docs/components/frame.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+sidebarDepth: 0
+---
+
+# Frame
+
+`<Frame>` is a UI component used to display `<Page>` elements.
+
+A frame with a default page
+
+```html
```
Comment (osmanertem) [inline docs/components/frame.md]: there should be a live demo section that lets user to modify all attributes of the component

File: docs/components/frame.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+},
+webViewLoadFinished(event) {
+  console.log("WebView Loading finished.");
+},
+```
+
+<FrameDoc />
+</DocExampleBox>
+
+## References
```
Comment (osmanertem) [inline docs/components/frame.md]: references section should be removed

File: package-lock.json [hunk 3]
```diff
@@ -14162,24 +14162,29 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "bundled": true,
+          "resolved": false,
```
Comment (osmanertem) [inline package-lock.json]: you should not push package-lock.json file unless you made changes on package.json

File: docs/components/frame.md (hunk 2 above)
Comment (legionairre) [reply docs/components/frame.md]: It has been removed

File: src/main.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ export { default as ActionBar } from './components/ActionBar';
 export { default as ActionItem } from './components/ActionItem';
 export { default as NavigationButton } from './components/NavigationButton';
 export { default as Label } from './components/Label';
+export { default as Frame } from './components/Frame';
```
Comment (zhunor) [inline src/main.js]: As we discussed avoid exports from main.js

File: tests/unit/components/Frame.spec.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import { mount } from '@vue/test-utils';
+import { Frame, Label } from '../../../src/main';
```
Comment (zhunor) [inline tests/unit/components/Frame.spec.js]: Please import components from their original files not from main.js. Avoid exports in main.js

File: docs/.vuepress/components/FrameDoc.vue [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<template>
```
Comment (mylmz10) [inline docs/.vuepress/components/FrameDoc.vue]: Could you please run lint for this file to fix indentation and other styling issues.

File: docs/components/frame.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    </GridLayout>
+  </Page>
+</Frame>
+```
+
+`Frames` can be multiple
+
+```html
+<Frame />
+    <TextView v-model="textViewInput" :editable="true" />
```
Comment (mylmz10) [inline docs/components/frame.md]: TextView should move into Frame?

File: src/main.js (hunk 4 above)
Comment (legionairre) [reply src/main.js]: export { default as Frame } from './components/Frame'; has been removed

File: tests/unit/components/Frame.spec.js (hunk 5 above)
Comment (legionairre) [reply tests/unit/components/Frame.spec.js]: Frame and label imported from the original path

File: docs/.vuepress/components/FrameDoc.vue (hunk 6 above)
Comment (legionairre) [reply docs/.vuepress/components/FrameDoc.vue]: It has been fixed

File: docs/components/frame.md (hunk 7 above)
Comment (legionairre) [reply docs/components/frame.md]: Actually TextView already inside in Frame but there was a wrong closed Frame tag. It has been fixed.

**Pull request merged**